### DISCUSSION TITLE
Is there an option to widen the space for the graph?

### DISCUSSION THREAD

**Discussion by alesalloum** [gephi/gephi#2797] [Q&A] [answered]
Hi,

Is there a setting somewhere, where I could make more space for the network in the "Graph"-tab? Now as you can see from the screenshot, the network kind of disappear from a invisible square when any layout-algorithm is used. 

![image](https://user-images.githubusercontent.com/50795403/229926432-9ab9ee97-8299-408b-bdce-5fa96d804752.png)

**Comment by mbastian** [comment, marked as answer]
Hi @alesalloum, yes you can configure that in settings. Increase the Max Size. You may have to restart Gephi after that.

<img width="421" alt="Screenshot 2023-04-05 at 08 36 58" src="https://user-images.githubusercontent.com/234813/230000425-2dfbf7c5-0538-4c69-bea1-7874453354d8.png">

**Reply by alesalloum** [reply]
Thanks you so much @mbastian. A few years back, the default was way bigger than the 50,000. Therefore, I thought it to be a bug first.
